xml/SectorCategory.xml

```diff
@@ -9,15 +9,24 @@
         </category>
     </metadata>
     <codelist-items>
+        <codelist-item status="active" activation-date="2019-11-10">
+            <code>110</code>
+            <name>
+                <narrative>Education</narrative>
+                <narrative xml:lang="fr">Education</narrative>
+            </name>
+            <description>
+                <narrative/>
+            </description>
+        </codelist-item>
```

Comment:
What’s going on here? I wonder why this was missing…

Reply:
I think this is a funny case where this code groups several other categories underneath it? So there are no 5-digit purpose codes that link directly to it, rather they sit under 111, 112, etc.

Reply:
Aha! Okay, that makes sense.

But does that mean it should be present, or not present?